api/command_test.go

```diff
@@ -184,3 +185,53 @@ func TestEchoCommand(t *testing.T) {
 		t.Fatal("Echo command failed to send")
 	}
 }
+
+func TestLoadTestUrlCommand(t *testing.T) {
+	Setup()
+
+	// enable testing to use /loadtest but don't save it since we don't want to overwrite config.json
+	utils.Cfg.ServiceSettings.EnableTesting = true
```

Comment:
I think you might need to to it back to false at the end of the test.  Some tests will save the config file.  So if this unit test is ran before those then it will modify the config.json.